examples/controls_example/public/react_controls/data_controls/options_list_control/components/options_list_popover_suggestions.tsx

```diff
@@ -196,7 +196,6 @@ export const OptionsListPopoverSuggestions = ({
           )}
           emptyMessage={<OptionsListPopoverEmptyMessage showOnlySelected={showOnlySelected} />}
           onChange={(newSuggestions, _, changedOption) => {
-            setSelectableOptions(newSuggestions);
```

Comment:
Removed because its not needed. `setSelectableOptions` when `useEffect` runs because of observable changes made when `makeSelection` fires